packages/ketcher-standalone/src/infrastructure/services/struct/standaloneStructService.ts

```diff
@@ -460,7 +490,10 @@ class IndigoService implements StructService {
         data: commandData
       }
 
-      worker.postMessage(inputMessage)
+      this.EE.removeListener('calculateCip', action)
```

Comment:
What do you think about replacing string literal with `enum` 
for example :
```js
 this.EE.removeListener(ListenerType.CalculateCip, action)
```

Reply:
Good proposal :) Will do